### ISSUE TITLE
Clarification on the use of the fixed value 217 in the code

### ISSUE THREAD

**Issue by MorningStarOvO** [stepfun-ai/Step1X-Edit#79]
<img width="732" alt="Image" src="https://github.com/user-attachments/assets/6c0a648a-7034-4147-976f-0812bdc42eab" />

Hello,

I hope you're doing well. I came across the following section of the code:

embs[idx, : min(self.max_length, emb.shape[1] - 217)] = emb[0, 217:][: self.max_length]
masks[idx, : min(self.max_length, emb.shape[1] - 217)] = torch.ones(
    (min(self.max_length, emb.shape[1] - 217)),
    dtype=torch.long,
    device=torch.cuda.current_device(),
)
I noticed that the value 217 is used in a few places, and I was wondering if you could provide some insight into why this specific value was chosen. Is it related to a particular aspect of the model, or is there a specific reason behind its selection?

请问这里的固定值 217 代表什么意思呢？是 prefix prompt 的 token 长度吗？没太理解 

Thank you so much for your help! I appreciate your time and any clarification you can offer.

**Comment by tingxueronghua** [comment]
Avoid processing the quotation within the system prompt.

**Comment by MorningStarOvO** [comment]
Hello,

Your explanation cleared up my doubts, and I now have a better understanding of its purpose.

I really appreciate your time and assistance in resolving this issue. Thanks again for your support!

Best regards,
